ide/coqide/wg_Find.ml

```diff
@@ -14,37 +14,39 @@ class finder name (view : GText.view) =
 
   let widget = Wg_Detachable.detachable
     ~title:(Printf.sprintf "Find & Replace (%s)" name) () in
-  let replace_box = GPack.grid (* ~columns:4 ~rows:2 *) ~col_homogeneous:false ~row_homogeneous:false
+  let replace_box = GPack.grid ~col_homogeneous:false ~row_homogeneous:false
     ~packing:widget#add () in
-  let hb = GPack.hbox ~packing:(replace_box#attach
-      ~left:1 ~top:0 (*~expand:`X ~fill:`X*)) () in
+  let hb = GPack.hbox ~packing:(replace_box#attach ~left:1 ~top:0) () in
   let use_regex =
     GButton.check_button ~label:"Regular expression"
       ~packing:(hb#pack ~expand:false ~fill:true ~padding:3) () in
   let use_nocase =
     GButton.check_button ~label:"Case insensitive"
       ~packing:(hb#pack ~expand:false ~fill:true ~padding:3) () in
-  let _ = GMisc.label ~text:"Find:" ~xalign:1.0
-    ~packing:(replace_box#attach
-     (*~xpadding:3 ~ypadding:3*) ~left:0 ~top:1 (*~fill:`X*)) () in
-  let _ = GMisc.label ~text:"Replace:" ~xalign:1.0
```

Comment:
The xaligns are not needed?

Reply:
No, because I’m using `set_halign` below, which looks more relevant. If using `~xalign:1.0` instead, labels are aligned to the right. If using none of them, labels are centered. According to https://docs.gtk.org/gtk3/property.Label.xalign.html `xalign` should correspond to a “labels size allocation,” but well, how it works is less clear to me than `halign`.